### PR TITLE
chore: Remove logsCustomSql feature flag.

### PR DESCRIPTION
**Pull request by Ziinc**
Removes the `logsCustomSql` feature flag, which should only be merged after ~2 weeks of prod usage. Feature still in prod testing phase.

### CODE REVIEW

File: studio/pages/project/[ref]/settings/logs/[type].tsx [hunk 1]
```diff
@@ -256,18 +255,16 @@ export const LogPage: NextPage = () => {
...
-                    <Typography.Text small={true} type="secondary">
-                      Custom queries are restricted to a {type === 'database' ? '2 hour' : '7 day'}{' '}
-                      querying window.
-                    </Typography.Text>
-                  </div>
-                )}
-              </Flag>
+              {isSelectQuery && (
+                <div className="flex flex-grow flex-row items-center gap-x-1">
+                  {/* // we don't have a slim Alert component yet */}
```
Comment (joshenlim) [inline studio/pages/project/[ref]/settings/logs/[type].tsx:260]: You can use the <InformationBox /> component for this? just pass the `title` prop

Comment (Ziinc) [reply studio/pages/project/[ref]/settings/logs/[type].tsx]: done, updated the InformationBox component as well

**Comment by Ziinc** [comment]
@joshenlim avail for re-review?

**Review by joshenlim — APPROVED** [review]

**Comment by joshenlim** [comment]
lgtm! would you mind doing a rebase first?
image

PRs to master it's recommended to do a rebase and merge
whereas PRs to studio from master we can just do a merge commit

**Pull request merged**